packages/atlas/src/api/hooks/video.ts

```diff
@@ -44,6 +47,20 @@ export const useFullVideo = (
   }
 }
 
+export const useFullVideoByUniqueInput = (
+  id: string,
+  opts?: QueryHookOptions<GetFullVideoQuery, GetFullVideoQueryVariables>
+) => {
+  const { data, ...queryRest } = useGetFullVideoQuery({
+    ...opts,
+    variables: { where: { id } },
+  })
+  return {
+    video: data?.videoByUniqueInput,
+    ...queryRest,
+  }
+}
```

Comment:
I don't like that we're using this type of hook only for videos, we're inconsistent when it comes to nfts and channels. We should use the same pattern of hook names.

What if we only keep the old hook (`useFulLVideo`) and add an optional param too it like `cached?: boolean`? Based on that, we either run the `GetFullVideos` to get the full and filtered data from the network, or if true, we run `GetFullVideo` with `cache-only` fetch policy? What do you think?

Reply:
Sounds good, however I don't think we need to define `fetch-policy`, it's `cache-first` by default, right?

Reply:
Sorry, I meant `cache-only`